src/WebJobs.Extensions.DurableTask/EntityScheduler/RequestMessage.cs

```diff
@@ -81,7 +82,19 @@ internal class RequestMessage
         public int Position { get; set; }
 
         [JsonIgnore]
-        public bool IsLockRequest => this.LockSet != null;
+        public bool IsLockRequest => this.LockSet != null;
+
+        /// <summary>
+        /// Parent trace context of this request message.
+        /// </summary>
+        [JsonProperty(PropertyName = "parentTraceContext", DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public DistributedTraceContext ParentTraceContext { get; set; }
```

Comment:
Should this be a nullable property?

```suggestion
        public DistributedTraceContext? ParentTraceContext { get; set; }
```